common/extensions/brave_extensions_client.cc

```diff
@@ -51,6 +52,15 @@ void BraveExtensionsClient::InitializeWebStoreUrls(
   ChromeExtensionsClient::InitializeWebStoreUrls(command_line);
 }
 
+bool BraveExtensionsClient::IsScriptableURL(const GURL& url,
+                                            std::string* error) const {
+  if (skus::IsSafeOrigin(url)) {
+    *error = "This site is protected and cannot be scripted.";
```

Comment:
need to make sure `error` is not null here

Reply:
fixed